Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -13061,6 +13090,42 @@ - (BOOL)shouldSuppressEditMenu
     return !!_suppressSelectionAssistantReasons;
 }
 
+#pragma mark - WKSTextStyleSourceDelegate
+
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+- (void)targetedPreviewForID:(NSUUID *)uuid completionHandler:(void (^)(UITargetedPreview *))completionHandler
+{
+    auto textUUID = WTF::UUID::fromNSUUID(uuid);
+    _page->getTextIndicatorForID(*textUUID, [protectedSelf = retainPtr(self), completionHandler = makeBlockPtr(completionHandler)] (std::optional<WebCore::TextIndicatorData> indicatorData) {
+
+        if (!indicatorData)
+            completionHandler(nil);
```

Comment:
can `nil` safety be converted to `std::nullopt`?

Reply:
Where is nil being converted to null opt?